supertokens_python/recipe/passwordless/interfaces.py

```diff
@@ -176,16 +176,19 @@ def __init__(self, status: Literal['OK', 'UNKNOWN_USER_ID_ERROR',
 class DeleteUserInfoResult(ABC):
     def __init__(self, status: Literal['OK', 'UNKNOWN_USER_ID_ERROR']):
         self.status = status
+        self.is_ok = False
 
```

Comment:
You also need a `self.is_unknown_user_id_error`. Again, I know it serves no purpose, but consistency. If you are confused about this, we can discuss on a call.